packages/components/fields/async-select-field/src/async-select-field.js

```diff
@@ -0,0 +1,360 @@
+import { Component } from 'react';
```

Comment:
You might want to correctly rebase your branch with `main`, as these components have already been migrated to TS.